doc/api.yml

```diff
@@ -258,6 +258,41 @@ paths:
           $ref: '#/components/responses/429'
         '500':
           $ref: '#/components/responses/500'
+  /students/{student_id}/grades:
+    get:
+      tags:
+        - Teaching
+      summary: | 
+        Get a list of courses. 
```

Comment:
Summary should be a SUMMARY. 

Put this inside the description if necessary and put a summary here instead.